tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Provision VM for simulator
+  ansible.builtin.import_role:
+    name: cloud.vmware_ops.provision_vm
+  tags: integration-ci
+
+- name: Perform VM Lifecycle Operations in vCenter Environment
+  block:
+  - name: Provision multiple VMs
+    loop: "{{ provision_vms }}"
+    ansible.builtin.include_tasks: provision_vms_with_validation.yml
+
+  - name: Turn one of the provisioned VMs into template
+    loop: "{{ provision_vms_template }}"
+    community.vmware.vmware_guest:
+      hostname: "{{ provision_vm_hostname }}"
+      username: "{{ provision_vm_username }}"
+      password: "{{ provision_vm_password }}"
+      validate_certs: "{{ provision_vm_validate_certs }}"
+      name: "{{ item.provision_vm_name }}"
+      is_template: true
+      folder: "{{ provision_vm_folder }}"
+      datacenter: "{{ provision_vm_datacenter }}"
+
+  - name: Create a VM from the template
+    loop: "{{ provision_vms_from_template }}"
+    ansible.builtin.include_tasks: provision_vms_with_validation.yml
+
+  - name: Update one the provisioned VMs
+    ansible.builtin.include_tasks: update_vm.yml
+
+  always:
```

Comment:
Here we should add a task to remove the network created in [this task](https://github.com/redhat-cop/cloud.vmware_ops/pull/53/files#diff-308cb569dd900a593a02620bce7b644c2ed748db334d802f61cc16492116022eR29-R37)

Reply:
Will it be part of that PR? 
If so please change the PR to [WIP]

Reply:
tnx Elsa! I haven't noticed the missing cleanup here
I added the task, if the CI pass, we can merge this PR